cif_rho.dic

```diff
@@ -2124,6 +2038,111 @@ save_atom_rho_multipole_radial_slater.n3_su
 
 save_
 
+save_atom_rho_multipole_radial_slater.n_list
+
+    _definition.id                '_atom_rho_multipole_radial_slater.n_list'
+    _alias.definition_id          '_atom_rho_multipole_radial_slater_n_list'
+    _definition.update            2024-04-23
+    _description.text
+;
+    These items are used when the radial dependence of the valence
+    electron  density, R(kappa'(l),l,r), of the atom specified in
+    atom_rho_multipole.atom_label is expressed as a Slater-type
+    function [Hansen & Coppens (1978), equation (3)]:
+
+    R(kappa'(l),l,r) = [{zeta(l)\}^\{n(l)+3\}^/\{n(l)+2\}!]\
+                        *(kappa'(l)*r)^n(l)^
+                        *exp(-kappa'(l)*zeta(l)*r)
+
+    where:
+      kappa'(l)  = atom_rho_multipole_kappa.prime[l]
+      n(l)       = atom_rho_multipole_radial_slater.n[l]
+      zeta(l)i   = atom_rho_multipole_radial_slater.zeta[l]
+
+    R(kappa'(l),l,r) appears in the multipole formalism described by
+    Hansen & Coppens [1978, equation (2)] which gives the
+    electron density at position vector r with respect to an
+    atomic nucleus as:
+
+    rho(r) = Pc*rho_core(r) + Pv*kappa^3^*rho_valence(kappa*r)
+            + sum{k'(l)^3^*R(kappa'(l),l,r)\}\
+              *sum{P(l,m)*d(l,m,theta,phi)\}\
+
+    where:
+      Pc     = atom_rho_multipole_coeff.Pc
```

Comment:
This definition and the definition of `_atom_rho_multipole_radial_slater.zeta_list` sometimes refers to data items without the leading underscore. It would be nice to homogenise the notation where possible.

Reply:
Have made a first pass at this. There may be others that will be spotted in a closer scrutiny later.